packages/bp/jest.e2e.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     }
   },
   globalSetup: 'jest-environment-puppeteer/setup',
-  setupFilesAfterEnv: ['expect-puppeteer', './jest.setup.js'],
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.ts'],
```

Comment:
should this be js ?

```suggestion
  setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
```

Reply:
I intentionally put `.ts` here since I want to be using typescript files everywhere I can. Also, since we are using `ts-jest` (see the transformer property), the file is normally transpiled on the fly!